contrib/docker/tester/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
```

Comment:
This will switch from gcc 5 to gcc 9. Not a problem, but this means we no longer test an older gcc. Thoughts?